auth0/resource_auth0_log_stream_test.go

```diff
@@ -0,0 +1,213 @@
+package auth0
+
+import (
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/alexkappa/terraform-provider-auth0/auth0/internal/random"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("auth0_log_stream", &resource.Sweeper{
+		Name: "auth0_log_stream",
+		F: func(_ string) error {
+			api, err := Auth0()
+			if err != nil {
+				return err
+			}
+			l, err := api.LogStream.List()
+			if err != nil {
+				return err
+			}
+			for _, logstream := range l {
+				if strings.Contains(logstream.GetName(), "Test") {
+					log.Printf("[DEBUG] Deleting logstream %v\n", logstream.GetName())
+					if e := api.LogStream.Delete(logstream.GetID()); e != nil {
+						multierror.Append(err, e)
+					}
+				}
+			}
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	})
+}
+
+func TestAccLogStreamHttp(t *testing.T) {
+	rand := random.String(6)
+
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]terraform.ResourceProvider{
+			"auth0": Provider(),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: random.Template(logStreamHTTPConfig, rand),
+				Check: resource.ComposeTestCheckFunc(
+					random.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "name", "Acceptance-Test-LogStream-http-{{.random}}", rand),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "type", "http"),
+					//resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "status", "active"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_endpoint", "https://example.com/webhook/logs"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_content_type", "application/json"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_content_format", "JSONLINES"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_authorization", "AKIAXXXXXXXXXXXXXXXX"),
+				),
+			},
```

Comment:
Additional tests to verify updates work as expected.

```suggestion
			},
			{
				Config: random.Template(testAccLogStreamHttpConfigUpdate, rand),
				Check: resource.ComposeTestCheckFunc(
					random.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "name", "Acceptance-Test-LogStream-http-{{.random}}", rand),
					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "type", "http"),
					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_endpoint", "https://example.com/webhook/logs"),
					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_content_type", "application/json; charset=utf-8"),
					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_content_format", "JSONARRAY"),
					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_authorization", "AKIAXXXXXXXXXXXXXXXX"),
				),
			},
```